xml/cl.xml

```diff
@@ -7290,11 +7327,11 @@ server's OpenCL/api-docs repository.
             <require>
                 <type name="CL/cl.h"/>
             </require>
-            <require comment="clGetEventInfo response when param_name is CL_EVENT_COMMAND_TYPE">
+            <require group="cl_command_type" comment="clGetEventInfo response when param_name is CL_EVENT_COMMAND_TYPE">
```

Comment:
I don't think the comment is adding much here, and it's inconsistent with the other enum blocks.  I'd be fine removing it.

```suggestion
            <require group="cl_command_type">
```